src/store/src/Bridge/Redis/Distance.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store\Bridge\Redis;
+
+use OskarStark\Enum\Trait\Comparable;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+enum Distance: string
+{
+    use Comparable;
+
+    case Cosine = 'COSINE';
+    case L2 = 'L2';
+    case Ip = 'IP';
+
+    public function getRedisMetric(): string
```

Comment:
```suggestion
    public function getMetric(): string
```

It is already in the Redis namespace

Reply:
or shall we use `->value` instead of a getter?